mapit_it/templates/mapit/base.html

```diff
@@ -1,88 +1,4 @@
-{% load static from staticfiles %}
+{% extends "mapit/base.html" %}
```

Comment:
The only reason I'm overriding `base.html` here is to insert the translated page description. Could we just upload the translation via Transifex instead, and then the override wouldn't even be required?